charts/argocd-image-updater/templates/rbac.yaml

```diff
@@ -26,23 +59,17 @@ rules:
       - list
       - update
       - patch
-  - apiGroups:
-      - ""
-    resources:
-      - events
-    verbs:
-      - create
```

Comment:
Do we need to keep those (in the "non-cluster" Role) in case someone uses `createClusterRoles: false` ?

Reply:
I think so too. 💡 
I added in https://github.com/argoproj/argo-helm/pull/2992/commits/c5a0eea9c1e1798481786a558c5eb95eef069a8f 🙋